src/component-select/component-select.scss

```diff
@@ -10,30 +10,32 @@
 
   button {
     margin-left: $spacing;
-
-    &:hover {
-      color: green;
-    }
   }
 
   .page-area {
     display: flex;
     align-items: center;
 
-    button {
+    .edit-page-name {
       background-color: transparent;
       border: none;
-      font-size: 20px;
+      font-size: 15px;
       padding: 0;
+      &:hover {
+        color: $primary-orange;
+      }
     }
 
-    .page {
+    .page, .page input[type='text'] {
```

Comment:
It seems like you are handling the input below on lines 36-38. Do you still need to change line 29?

Reply:
This was so both .page and .page input would share all of these styles, minimizing replication. Then I broke our what was specific to the input on 36. Im open to a better way if you see one.